### PR TITLE
feat: support ranges for Value Mappings

### PR DESCRIPTION
**Pull request by Fine0830**
Related https://github.com/apache/skywalking/pull/12696#issuecomment-2413085299

Video

https://github.com/user-attachments/assets/20247225-cdf9-4527-b308-0485cebeb444

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

File: src/views/dashboard/graphs/Card.vue [hunk 1]
```diff
@@ -48,18 +48,45 @@ limitations under the License. -->
...
+      const k = i.replace(/\s+/g, "");
+      let withinRange = false;
+      const ranges = k.match(regex)?.map(Number) || [];
+      if (k.startsWith("[")) {
+        withinRange = k.startsWith("[-∞") || Number(singleVal.value) >= ranges[0];
+      } else {
+        withinRange = k.startsWith("(-∞") || Number(singleVal.value) > ranges[0];
+      }
+      if (k.endsWith("]")) {
+        withinRange = withinRange && (k.endsWith("+∞]") || Number(singleVal.value) <= (ranges[1] || ranges[0]));
```
Comment (wu-sheng) [inline src/views/dashboard/graphs/Card.vue:80]: So, `+∞` and `-∞` are supported if we need no upper or lower boundary?

**Comment by wu-sheng** [comment]
image

Could you we add tips about how to setup the range? The contributors/users will not be able to watch this video and learn how to use this. We need to way to guide them.

Or

If you want to put this in the documents, you should update this doc. https://skywalking.apache.org/docs/main/next/en/ui/readme/

**Comment by wu-sheng** [comment]
fyi @CodePrometheus

File: src/views/dashboard/graphs/Card.vue (hunk 1 above)
Comment (Fine0830) [reply src/views/dashboard/graphs/Card.vue:80]: Yep.

Comment (wu-sheng) [reply src/views/dashboard/graphs/Card.vue:80]: I have a little concern of `∞`, it is hard to input from the keyboard. How about `infinite+` and `infinite-`

Comment (CodePrometheus) [reply src/views/dashboard/graphs/Card.vue:80]: If this supports writing regular expressions directly, how about that. For example, ^(10|[0-9])$ matches 0-10.

Comment (CodePrometheus) [reply src/views/dashboard/graphs/Card.vue:80]: `\b(2[5-9]|3[0-4])\b` could catch up, regex may not be as intuitive as the current method.

Comment (wu-sheng) [reply src/views/dashboard/graphs/Card.vue:80]: @Fine0830 Could you try to change the format to include a regex?

**Comment by wu-sheng** [comment]
@CodePrometheus I think we need to update your dashboard to adopt this change. Could you do that soon? I could merge this when you are ready to follow.

FYI, the config key is changed, so you need to remove the old one and add the new config.

**Review by CodePrometheus — APPROVED** [review]

**Comment by CodePrometheus** [comment]
> @CodePrometheus I think we need to update your dashboard to adopt this change. Could you do that soon? I could merge this when you are ready to follow.
> 
> FYI, the config key is changed, so you need to remove the old one and add the new config.

Let me sync ui later, thanks @Fine0830

**Pull request merged**